Core/Core_c.php

```diff
@@ -795,7 +795,7 @@ public function offsetUnset($object): void {}
     /**
      * Returns an iterator in the "[object => mixed]" format.
      *
-     * @return Traversable<TKey, TValue>
+     * @return Iterator
```

Comment:
Traversable<TKey, TValue> was using templates intentionally. So the new Type also should have them